src/boot/stylesheets/main.scss

```diff
@@ -39,20 +39,18 @@ body {
     	color: $blue-wordpress;
     }
 
-    a[disabled], .wpnc__single-view header .back[disabled] {
-    	pointer-events: none;
-    	color: $gray;
-    }
-
     button {
 		background-color: transparent;
 		border: none;
+		color: $blue-wordpress;
 		cursor: pointer;
 		font-size: inherit;
+		outline: none;
 		padding: 0;
 
-		&.focus {
-			outline: none;
+		&[disabled] {
+			pointer-events: none;
```

Comment:
How about `cursor: default`? Using`pointer-events:none` is rather unusual and has some quirks in IE11.